CUDA buld is broken (videostab)

[Nightly build](http://pullrequest.opencv.org/buildbot/builders/master-contrib_cuda-lin64/builds/202).

```
In file included from /build/master-contrib_cuda-lin64/opencv_contrib/modules/videostab/src/deblurring.cpp:45:0:
/build/master-contrib_cuda-lin64/opencv_contrib/modules/videostab/include/opencv2/videostab/global_motion.hpp:287:5: error: 'GpuMat' does not name a type
     GpuMat mask_;
     ^
/build/master-contrib_cuda-lin64/opencv_contrib/modules/videostab/include/opencv2/videostab/global_motion.hpp: In member function 'virtual void cv::videostab::KeypointBasedMotionEstimatorGpu::setFrameMask(cv::InputArray)':
/build/master-contrib_cuda-lin64/opencv_contrib/modules/videostab/include/opencv2/videostab/global_motion.hpp:277:62: error: 'mask_' was not declared in this scope
     virtual void setFrameMask(InputArray mask) CV_OVERRIDE { mask_ = mask.getMat(); }
                                                              ^
```

relates #2052

--- a/modules/videostab/include/opencv2/videostab/global_motion.hpp
+++ b/modules/videostab/include/opencv2/videostab/global_motion.hpp
@@ -274,8 +274,6 @@ public:
     void setOutlierRejector(Ptr<IOutlierRejector> val) { outlierRejector_ = val; }
     Ptr<IOutlierRejector> outlierRejector() const { return outlierRejector_; }
 
-    virtual void setFrameMask(InputArray mask) CV_OVERRIDE { mask_ = mask.getMat(); }
-
     virtual Mat estimate(const Mat &frame0, const Mat &frame1, bool *ok = 0) CV_OVERRIDE;
     Mat estimate(const cuda::GpuMat &frame0, const cuda::GpuMat &frame1, bool *ok = 0);
 
@@ -284,7 +282,6 @@ private:
     Ptr<cuda::CornersDetector> detector_;
     SparsePyrLkOptFlowEstimatorGpu optFlowEstimator_;
     Ptr<IOutlierRejector> outlierRejector_;
-    GpuMat mask_;
 
     cuda::GpuMat frame0_, grayFrame0_, frame1_;
     cuda::GpuMat pointsPrev_, points_;

--- a/modules/videostab/src/global_motion.cpp
+++ b/modules/videostab/src/global_motion.cpp
@@ -815,7 +815,7 @@ Mat KeypointBasedMotionEstimatorGpu::estimate(const cuda::GpuMat &frame0, const 
     }
 
     // find keypoints
-    detector_->detect(grayFrame0, pointsPrev_, mask_);
+    detector_->detect(grayFrame0, pointsPrev_);
 
     // find correspondences
     optFlowEstimator_.run(frame0, frame1, pointsPrev_, points_, status_);

